### PR TITLE
PBRP: Device Names will include vendor names

### PR DESCRIPTION
**Pull request by manjotsidhu**
It's been so long that our Telegram Releases were having vendor starting with lowercase character as it was `codename`. Now we will use the full device name including vendor name inside our `pb_devices.json`

[**NOT TESTED**] Please test and merge if everything seems fine.

### CODE REVIEW

File: pb_deploy.sh [hunk 1]
```diff
@@ -87,7 +87,7 @@ if [[ "$zipcounter" > "0" ]]; then
 				link="https://sourceforge.net/projects/pbrp/files/${NAME}/$(echo $sf_file | awk -F'[/]' '{print $NF}')"
 				curl -i -X POST 'https://us-central1-pbrp-prod.cloudfunctions.net/release' -H "Authorization: Bearer ${GCF_AUTH_KEY}" -H "Content-Type: application/json" --data "{\"codename\": \"$codename\", \"vendor\":\"$VENDOR\", \"md5\": \"$MD5\", \"size\": \"$file_size\", \"sf_link\": \"$link\", \"gh_link\": \"$gh\",\"version\": \"$pbv\"}"
 				link="https://pitchblackrecovery.com/$(echo $codename | sed "s:_:-:g")"
-				FORMAT="PitchBlack Recovery for <b>$VENDOR $TARGET_DEVICE</b> (<code>${NAME}</code>)\n\n<b>Info</b>\n\nPitchBlack V${pbv} Official\nBased on TWRP ${TWRP_V}\n<b>Build Date</b>: <code>${build:0:4}/${build:4:2}/${build:6}</code>\n\n<b>Maintainer</b>: ${maintainer}\n"
```
Comment (AndroiableDroid) [inline pb_deploy.sh:90]: so where will rthe vendor or oem will come?

Comment (manjotsidhu) [reply pb_deploy.sh:90]: Device name in json already includes oem aka Full Device Name.

**Comment by AndroiableDroid** [comment]
Am asking for release in the channel

File: pb_deploy.sh (hunk 1 above)
Comment (AndroiableDroid) [reply pb_deploy.sh:90]: https://github.com/PitchBlackRecoveryProject/android_device_redmi_begonia-pbrp/blob/android-9.0/.circleci/config.yml#L9

**Comment by manjotsidhu** [comment]
Ok got it

**Pull request merged**